base/interactiveutil.jl

```diff
@@ -373,9 +376,11 @@ end
     filename
 end
 
-function download(url::AbstractString)
+function download(url::AbstractString; silent::Bool=true)
     filename = tempname()
-    download(url, filename)
+    @unix_only download(url, filename, silent=silent)
+    @windows_only download(url, filename)
+    filename
```

Comment:
I'd remove this. Put a return on the `download` line if you want things to be clearer